cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -0,0 +1,255 @@
+"""Test to validate basic navigations.Later to be replaced with End-to-End functional testing."""
```

Comment:
Tests from this file are already in #7498  here I have just removed the common fixtures from here and put it under cfme/fixtures/v2v.py, as @Yadnyawalkya did in #7540